lib/tasks/wagons.rake

```diff
@@ -192,6 +192,16 @@ namespace :db do
       if migrated.size > context.migrations.size
         puts "The database schema will not be dumped when there are loaded wagon migrations."
         puts "To dump the application schema, please 'rake wagon:remove WAGON=ALL' wagons beforehand or reset the database."
+
+        schema = Rails.root.join("db/schema.rb")
+        tmp_schema = Rails.root.join("tmp/schema.rb")
+        wagon_schema = Wagons.all.first.root.join("db/schema.rb")
+        wagon_schema_diff = Wagons.all.first.root.join("db/schema.diff")
```

Comment:
Als Library Entwickler ist `Wagons.all.first` ein Stich ins Herz. Das mag ja bei hitobito funktionieren, ist aber überhaupt nicht generalisierbar. Es gibt keinen allgemeinen "ersten" Wagon, welcher das endgültige Applikationsschema zusammenbringt. Wagons erlauben ebenfalls beliebige parallele "Feature-Wagons", welche einzelne, unabhängige Migrationen enthalten können. Daher macht so ein Dump in der Gem Funktionalität imho keinen Sinn. Ich würde den entsprechenden Code direkt in hitobito einbauen, könnte ja durchaus ein eigener Rake Task sein.